nvdaHelper/vbufBase/backend.cpp

```diff
@@ -292,6 +287,11 @@ VBufStorage_controlFieldNode_t* VBufBackend_t::reuseExistingNodeInRender(VBufSto
 		LOG_DEBUG(L"Existing node and its descendants must not be reused");
 		return nullptr;
 	}
+	// Ensure the node allows us to reuse it

```

Comment:
considering the importance of the order of execution, is it possible to add a comment explaining why the location of this snippet is important. e.g. "ensure alwaysRerenderDescendants is propagated before returning early".